todo_app/app.py

```diff
@@ -1,38 +1,45 @@
 from flask import abort, Flask, render_template, request, redirect, session, url_for
-from todo_app.data.session_items import add_item, delete_item, get_item, get_items, save_item
+from todo_app.data.trello_api import Trello
 from todo_app.flask_config import Config
+from os import environ
+
+BOARD_KEY = environ.get('BOARD_KEY')
+TRELLO_KEY = environ.get('TRELLO_KEY')
+TRELLO_TOKEN = environ.get('TRELLO_TOKEN')
```

Comment:
We should add some warning to our Readme, and add these keys to the `.env.template` file, so that other developers know what settings they need to set